File: TYPESCRIPT PROGRAMMING/Test47.ts
// 47. FACTORIAL OF A NUMBER
const factorial = (ip: number): number => {
    let temp: number = 1;
    for(let i: number = 1; i <= ip; i++) {
        temp *= i;
    }
    return temp;
}
console.log(`Factorial Of 5: ${factorial(5)}`);
console.log(`Factorial Of 6: ${factorial(6)}`);
console.log(`Factorial Of 10: ${factorial(10)}`);

File: JAVA PROGRAMMING/Test47.java
// 47. FACTORIAL OF A NUMBER
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Input Value: ");
        int ip = in.nextInt();
        int temp = 1;
        for(int i = 1; i <= ip; i++) {
            temp *= i;
        }
        System.out.println("Factorial Of "+ip+" Is: "+temp);

        in.close();
    }
}